pytest_fixtures/component/provision_pxe.py

```diff
@@ -118,7 +119,47 @@ def module_provisioning_rhel_content(
         ],
     ).create()
 
-    return Box(hostgroup=hostgroup, os=os)
+    return Box(hostgroup=hostgroup, os=os, ak=ak)
+
+
+@pytest.fixture(scope='module')
+def module_provisioning_client_repo(
+    request,
+    module_provisioning_rhel_content,
+    module_org_with_manifest,
+    module_provisioning_sat
+):
+    """
+    This fixture sets up kickstart repositories for a specific RHEL version
+    that is specified in `request.param`.
+    """
+    sat = module_provisioning_sat.sat
+    rhel_ver = request.param['rhel_version']
+    # add sat client repo
+    client_repo = (
+        f'{settings.repos["DOGFOOD_REPO_HOST"].replace("http", "https")}/pulp/content/'
+        'Satellite_Engineering/QA/Satellite_Client/custom/Satellite_Client_Composes/'
+        f'Satellite_Client_RHEL{rhel_ver}_x86_64/'
+    )
+    # TODO client_repo should be changed to
+    # settings.repos['SATCLIENT_REPO'][f'RHEL{rhel_ver}']
+    # when/if the new dogfood url pattern settles
```

Comment:
I think you can now use `settings.repos['SATCLIENT_REPO'][f'RHEL{rhel_ver}']` here

Reply:
not really, if you try to sync from that, you'll get 404

Reply:
With the patch from #10195, you can do
```python
client_repo = sat.dogfood_repository(product='client', repo='client', release='Client')
product = sat.cli.make_product({'organization-id': module_org_with_manifest.id})
sat.cli.make_repository(
    {
        'name': client_repo.name,
        'product-id': product['id'],
        'organization-id': module_org_with_manifest.id,
        'content-type': 'yum',
        'url': client_repo.baseurl,
    }
)
```

Reply:
Using #10195 in the latest revision, your code can look like:
```python
from robottelo.utils.ohsnap import dogfood_repository
client_repo = dogfood_repository(
    settings.repos.ohsnap_repo_host,
    product='client',
    repo='client',
    release='Client',
    os_release=rhel_ver,
)
product = sat.cli.make_product({'organization-id': module_org_with_manifest.id})
sat.cli.make_repository(
    {
        'name': client_repo.name,
        'product-id': product['id'],
        'organization-id': module_org_with_manifest.id,
        'content-type': 'yum',
        'url': client_repo.baseurl,
    }
)
```

Question: why don't you use the following?
```
client_repo_url = settings.repos.satclient_repo.get(f'rhel{rhel_ver}')
```

Reply:
I see I have repeated the first question here in the thread. @pondrejk Then I would say that satelliteqe-jenkins !817 will fix the 404 issue.